.circleci/run.sh

```diff
@@ -163,8 +163,9 @@ check_clean_git()
     # Remove temporary directory + install script
     rm -rf _generated
     rm -f install.sh
-    # auto-removal of this file doesn't work on CirleCi
+    # auto-removal of these files doesn't work on CirleCi
     rm -f test/compilable/vcg-ast.d.cg
+    rm -f test/test.mixin
```

Comment:
This was meant as a safe guard to check that the testsuite didn't modify any checked-in files nor created new ones which aren't in test_results.

Reply:
https://github.com/dlang/dmd/pull/9236